addons/centered/src/html.js

```diff
@@ -0,0 +1,39 @@
+import { document, Node } from 'global';
+
+const style = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'auto',
+};
```

Comment:
Can we share this object between frameworks?

Reply:
Yeah, makes sense